core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3490,6 +3490,17 @@ public String toString() {
               + "format pattern.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_DIRECT_MEMORY_IO_ENABLED =
+      new Builder(Name.USER_DIRECT_MEMORY_IO_ENABLED)
```

Comment:
you are correct. any good naming suggested?

Reply:
It can be read from the only tier no matter it's memory or SSD/HDD. `USER_DIRECT_READ_ENABLED`? would be better to differentiate this property fro the USER_SHORT_CIRCUIT_ENABLED